.travis.yml

```diff
@@ -1,18 +1,35 @@
-# Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
+
 os:
   - linux
+
 julia:
-  - 1.0
```

Comment:
@ahojukka5 I think we should keep 1.0 testing always. That is Julia's LTS version. Or are you using a feature that requires 1.2 minimum?

Reply:
There's no anything 1.2 specific. I have been doing 1.2, 1.3, nightly in other packages. I'm not 100 % sure is this the way how it should be done.

Reply:
I would do: 1.0, 1.2, nightly. Following the logic that LTS and the latest released are always tested. Nightly is nice to have with allowing failures, because you can get early warning of the coming breaking changes.

Of course now 1.3 makes more sense than 1.2.

Reply:
I have been looking closely how they do in Documenter.jl, https://github.com/JuliaDocs/Documenter.jl/blob/master/.travis.yml, I didn't understand the point why they keep 1.0 but now it starts to make sense.

Reply:
@ahojukka5 read this: https://julialang.org/blog/2019/08/release-process